app/mock/environment/__init__.py

```diff
@@ -8,7 +8,7 @@
 from astropy.time import Time
 from astropy.units import Quantity
 
-from common.minimodel import Site, Variant, CloudCover, ImageQuality
+from app.common.minimodel import Site, Variant, CloudCover, ImageQuality
 
 
 class Env:
```

Comment:
Here's a move I'm not sure about... why should `mock` be part of `app`?

Reply:
Mock at the end should be removed in favor of `services` right?

Reply:
I think so... I don't think we'll need to keep `mock` if we have `services` that are actually implemented.